addon/styles/addon.scss

```diff
@@ -14,6 +14,8 @@ $margin: 1px; // Note: margin + border width equals border width of focus state
 $padding: 0 19px 0 6px;
 
 .frost-text {
+  display: flex;
+  align-items: center;
```

Comment:
Can you run this through cross-browser tests?  FireFox has been odd with flex before